main/heart_stokes.cc

```diff
@@ -21,6 +23,15 @@
                                             "piDoMUS", \
                                             interface); \
   ParameterAcceptor::initialize(prm_file, pde_name+"_used.prm"); \
+  pidomus.lamdas.output_step = [&] (???)           \
```

Comment:
What are the correct parameters here? 
The ones in `pidomus_lambdas.h` line 94?